packages/utils/src/fallbackSubgraph.ts

```diff
@@ -85,28 +87,30 @@ export class FallbackSubgraph<T extends SubgraphSdk> {
       const split = url.split("/");
       return split[split.length - 1];
     };
-    this.subgraphs = sdks.map(({ client, url }) => ({
-      url,
-      client,
-      record: {
-        // Typically used for logging, distinguishing between which subgraph is which, so we can monitor
-        // which ones are most in sync.
-        uri: getHostnameFromRegex(url) ? getHostnameFromRegex(url)!.split(".").slice(0, -1).join(".") : url,
-        name: getSubgraphName(url),
-        synced: true,
-        latestBlock: -1,
-        syncedBlock: -1,
-        lag: 0,
-        errors: undefined,
-      },
-      priority: 0,
-      metrics: {
-        calls: [],
-        cps: 0,
-        reliability: 0,
-        avgExecTime: 0,
-      },
-    }));
+    this.subgraphs = sdks.map(({ client, url }) => {
+      return {
+        url,
+        client,
+        record: {
+          // Typically used for logging, distinguishing between which subgraph is which, so we can monitor
+          // which ones are most in sync.
+          uri: getHostnameFromRegex(url) ? getHostnameFromRegex(url)!.split(".").slice(0, -1).join(".") : url,
```

Comment:
lol i know this isn't your code but can we please change this check's redundant method call?
`uri: url ? getHostnameFromRegex(url).split(".").slice(0, -1).join(".") : url`